queries/cmake/highlights.scm

```diff
@@ -1,3 +1,9 @@
+(normal_command
+  (identifier)
+  (argument (unquoted_argument)) @constant
+  (#match? @constant "^[A-Z][A-Z_]+$")
```

Comment:
Does this work also with lua-match?

Reply:
I am not sure what you mean

Reply:
`lua-match?` uses the Lua implementation fro regular expressions which is more efficient than vim's `match?`. `match?` supportsore features than Lua patterns

Sorry! I forgot about this comment. The PR looks great apart from that minor considation

Reply:
oh, I have no idea about `lua-match?`, I will try it out.